src/ResourceManager/Common/Commands.Common.Strategies/AsyncOperationVisitor.cs

```diff
@@ -0,0 +1,55 @@
+﻿using System.Collections.Concurrent;
```

Comment:
@sergey-shandar all files need the Microsoft license header

Reply:
Done.